docs/Deploying/Configuration.md

```diff
@@ -310,16 +310,13 @@ This setting should not usually be used.
 ##### Example YAML
 
 ```yaml
-apiVersion: v1
-kind: config
-spec:
-  msyql:
-    address: localhost:3306
-    database: fleet
-    password: fleet
-    max_open_conns: 50
-    max_idle_conns: 50
-    conn_max_lifetime: 50
+msyql:
```

Comment:
Typo: `mysql` not `msyql`

Reply:
@smaddock Thanks for catching that! Fixed.